prov/kubernetes/execution.go

```diff
@@ -355,14 +354,11 @@ func (e *execution) manageServiceResource(ctx context.Context, clientset kuberne
 		}
 
 		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelDEBUG, e.deploymentID).Registerf("k8s Service %s created in namespace %s", service.Name, namespace)
-
-		kubConf := e.cfg.Infrastructures["kubernetes"]
-		kubMasterIP := kubConf.GetString("master_url")
-		u, _ := url.Parse(kubMasterIP)
-		h := strings.Split(u.Host, ":")
+		node, _ := getHealthyNode(clientset)
```

Comment:
Ignoring errors is a bad practice...
You should at least log it (better is to emit an event log that would be seen by the user).
Errcheck + log  does not prevent to continue anyway with an empty string.